### PR TITLE
docs: update Vault Helm to 0.9.0

### PR DESCRIPTION
**Pull request by jasonodonnell**
Vault Helm 0.9.0 was just released so I added the new configurables and updated relevant documentation.

### CODE REVIEW

**Review by tvoran — COMMENTED** [review]
Added some suggestions.

I also see that the injector affinity default is no longer `null`, so we should probably update that in the docs as well.

**Comment by jasonodonnell** [comment]
@tvoran Good call, updated the affinity default for the injector. Thanks for the suggestions 👍

File: website/content/docs/platform/k8s/helm/configuration.mdx [hunk 1]
```diff
@@ -134,16 +169,32 @@ and consider if they're appropriate for your deployment.
       "sample/annotation2": "bar"
     ```
 
-  - `failurePolicy` (`string : ""`) - When set to `Fail`, an error calling the webhook causes the admission to fail and the API request to be rejected.
+  - `failurePolicy` (`string: "Ignore"`) - When set to `Fail`, an error calling the webhook causes the admission to fail and the API request to be rejected. When set to `Ignore`, an error calling the webhook is ignored and the API request is allowed to continue. Allowed values: `Fail`, `Ignore`.
```
Comment (tvoran) [inline website/content/docs/platform/k8s/helm/configuration.mdx:172]: Oops, the last sentence already exists on the next line:

```suggestion
  - `failurePolicy` (`string: "Ignore"`) - When set to `Fail`, an error calling the webhook causes the admission to fail and the API request to be rejected.
```

**Review by tvoran — APPROVED** [review]
One small oops.

**Pull request merged**